packages/core/src/Actor.ts

```diff
@@ -185,15 +193,15 @@ export class Actor<TEvent extends EventObject, TEmitted> {
     }
   }
   public receiveSignal(signal: LifecycleSignal) {
```

Comment:
shouldn't this be unified with `receive` as well?

Reply:
Done